### ISSUE TITLE
Search behaviour difference between main page and /app route

### ISSUE THREAD

**Issue by fridde** [MehediH/winstall#94]
Weirdly the search results for the search window on ```https://winstall.app/``` and ```https://winstall.app/apps``` yield different results.

When looking for the package Handbrake (which I knew exists) I got 0 results when using the search bar on ```/apps``` and 1 result on the main page.

The same is true when I directly use my browser search shortcut using the ```q```-parameter: ```https://winstall.app/apps?q=handbrake``` or ```https://winstall.app/?q=handbrake``` are not equal.

This is pretty unexpected behaviour.

**Comment by ehawman-rosenberg** [comment]
Yeah this ticket really should be titled "bizarre search behavior". I've noticed the whole "don't give too much information" behavior many, many times. Can't really trust the results at the moment.

**Comment by st-jliu** [comment]
Dear fridde:

I belive this issue already be fixed, please check

https://winstall.app/?q=handbrake
https://winstall.app/apps?q=handbrake

And this project was taken over by Splashtop now, we will continue the development and maintenance, you are welcome to provide further feedback and suggestions.
